packages/frontend/src/components/UserSettingsModal/InvitesPanel.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+    useCreateInviteLinkMutation,
+    useRevokeInvitesMutation,
+} from '../../hooks/useInviteLink';
+import { useTracking } from '../../providers/TrackingProvider';
+import { EventName } from '../../types/Events';
+
+const InvitePanel: FC = () => {
+    const { track } = useTracking();
+    const { showToastSuccess } = useApp();
+    const inviteLink = useCreateInviteLinkMutation();
+    const revokeInvitesMutation = useRevokeInvitesMutation();
+
+    return (
+        <div
+            style={{ height: '100%', display: 'flex', flexDirection: 'column' }}
+        >
+            <FormGroup
+                label="Invite users to your organization"
+                labelFor="invite-link-input"
+            >
+                {inviteLink.data ? (
+                    <>
+                        <InputGroup
+                            id="invite-link-input"
+                            type="text"
+                            readOnly
+                            value={`${window.location.protocol}//${window.location.host}/invite/${inviteLink.data.inviteCode}`}
+                            rightElement={
+                                <CopyToClipboard
+                                    text={`${window.location.protocol}//${window.location.host}/invite/${inviteLink.data.inviteCode}`}
+                                    options={{ message: 'Copied' }}
+                                    onCopy={() =>
+                                        showToastSuccess({
+                                            title: 'Invite link copied',
+                                        })
+                                    }
+                                >
+                                    <Button minimal icon="clipboard" />
+                                </CopyToClipboard>
+                            }
+                        />
+                        <Callout intent="primary" style={{ marginTop: 10 }}>
+                            Share this link with your colleagues and they can
+                            join your organization. This link will expire at{' '}
+                            <b>{formatTimestamp(inviteLink.data.expiresAt)}</b>
+                        </Callout>
+                    </>
+                ) : (
+                    <Button
+                        text="Create invite link"
+                        style={{ marginTop: 10 }}
+                        loading={inviteLink.isLoading}
+                        onClick={() => {
+                            track({
+                                name: EventName.INVITE_BUTTON_CLICKED,
+                            });
+                            inviteLink.mutate();
+                        }}
+                    />
+                )}
+            </FormGroup>
+            <Callout intent="warning" style={{ marginTop: 20 }}>
+                <p>
+                    This action will remove all active and expired invites
+                    links.
```

Comment:
Copy is a bit strange because the user won't expect the backend to keep the expired links.

Instead:

* This action will make all existing invites invalid
* Existing invites will no longer be valid if you revoke them.
* This action will revoke all existing invites.